source/guides/ssl-setup-nginx.rst

```diff
@@ -20,15 +20,33 @@ First, let's install nginx.
 
          sudo apt-get update
          sudo apt-get install nginx
+
+   .. tab:: CentOS
+
+      .. code-block:: bash
+
+         sudo yum install epel-release
+         sudo yum install nginx
          
 Now navigate to http://yourip. You should see a default nginx page.  
 
 Now set up an nginx virtual host. This will allow nginx to serve PufferPanel.
-Create and open a config file at /etc/nginx/sites-enabled/pufferpanel.conf
+Create and open a config file at /etc/nginx/sites-enabled/pufferpanel.conf or /etc/nginx/conf.d/ on CentOS
```

Comment:
I'd suggest changing it to something like the following to make it a little more clear what is for what system and to avoid less knowledgable users asking what the file should be named on CentOS:
```suggestion
Create and open a config file at /etc/nginx/sites-enabled/pufferpanel.conf on Ubuntu/Debian or /etc/nginx/conf.d/pufferpanel.conf on CentOS
```